config/application.rb

```diff
@@ -15,6 +15,8 @@ class Application < Rails::Application
     config.cache_max_age = ENV['CACHE_MAX_AGE'] || 10.seconds
 
     config.middleware.use Rack::BounceFavicon
+    require 'bounce_browserconfig'
```

Comment:
Not without pushing `lib` onto `$LOAD_PATH` first, which is what happens on line 13 with `config.autoload_paths << Rails.root.join('lib')`